cluster/node-pools/worker-default/userdata.yaml

```diff
@@ -87,3 +87,19 @@ write_files:
           cacheAuthorizedTTL: "5m"
           cacheUnauthorizedTTL: "30s"
       readOnlyPort: 10255 # TODO: remove after complete rollout of webhook auth
+
+  - owner: root:root
+    path: /etc/sysctl.d/02-vm-max-mmaps.conf
+    content: |
+      vm.max_map_count=524288
```

Comment:
could we add a comment or at least extend the PR description with the reason we choose this value? I'm not arguing that we should NOT have this value, but it should just be clear how/why the decision was made when we go back to this change in 6 months :)

Reply:
Updated the PR description.